app/classifier.py

```diff
@@ -198,14 +198,14 @@ def _classify_probs(self, seqs: List[str]) -> List[Dict[str, float]]:
         # TODO(bdittes): Use prediction['attention']
         self.logger.debug(probabilities)
         topic_probs: List[Dict[str, float]] = [{}] * len(seqs)
-        for i, seq in enumerate(seqs):
+        for i in range(len(seqs)):
             # map results back to topic strings, according to classifier metadata
             # e.g. {'education': 0.8, 'rights of the child': 0.9}
             topic_probs[i] = {
                 t: p
                 # p.item() is used to convert from numpy float to python float.
                 for t, p in zip(self.vocab, [p.item() for p in probabilities[i]])
-                if p >= 0.01
+                if p >= 0.001
```

Comment:
Make this a global to make it more obvious where we should change later if we feel the need?